crates/extrinsics/src/pallet_contracts_primitives.rs

```diff
@@ -0,0 +1,186 @@
+// Copyright 2018-2024 Parity Technologies (UK) Ltd.
+// This file is part of cargo-contract.
+//
+// cargo-contract is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// cargo-contract is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with cargo-contract.  If not, see <http://www.gnu.org/licenses/>.
+
+use pallet_contracts_uapi::ReturnFlags;
```

Comment:
It seems that we only need `pallet-contracts-uapi` for one flag. Maybe we should just copy it?